target/linux/octeon/base-files/lib/upgrade/platform.sh

```diff
@@ -87,6 +92,9 @@ platform_do_upgrade() {
 	itus,shield-router)
 		kernel=ItusrouterImage
 		;;
+	ubnt,edgerouter-4)
+		kernel=mmcblk0p1
+		;;
```

Comment:
This can be merged with er?

Reply:
yes, seems like it could be.

thanks, merged it with `er`.